query-engine/connectors/sql-query-connector/src/database/operations/write.rs

```diff
@@ -66,6 +136,7 @@ pub async fn create_record(
         },
     };
 
+    // TODO: fix dbgenerated
```

Comment:
I think this comment needs more explanation

Reply:
yep, I actually have to remove it :D it is already done. :+1: